examples/resource_partitioner/async_customization.cpp

```diff
@@ -6,6 +6,7 @@
 #include <hpx/include/parallel_executors.hpp>
 #include <hpx/runtime/threads/executors/default_executor.hpp>
 #include <hpx/runtime/threads/executors/pool_executor.hpp>
+#define GUIDED_EXECUTOR_DEBUG 1
```

Comment:
Should this be left in? Or is it an oversight?

Reply:
That one is deliberate, the example/test overrides the debug setting to dump out useful information.